packages/v4-sdk/src/functions/bin/getAmounts.test.ts

```diff
@@ -0,0 +1,95 @@
+import { WNATIVE } from '@pancakeswap/sdk'
+import { maxUint128 } from 'viem'
+import { beforeEach, describe, expect, test } from 'vitest'
+import { TEN_PERCENT_FEE } from '../../constants/fee'
+import { getAmounts } from './getAmounts'
+
+// const getRandomFee = () => {
+//   const random = Math.random()
+//   return BigInt(Math.floor(random * Number(TEN_PERCENT_FEE)))
+// }
+
+// const getRandomActiveId = () => {
+//   const random = Math.random()
+//   return BigInt(Math.floor(random * Number(maxUint24)))
+// }
+const getRandom = (max: number, min = 0): bigint => {
+  const random = Math.random()
+  return BigInt(Math.floor(random * (max - min) + min))
+}
+
+const createBinPool = (swapFee: bigint, activeId: bigint, reserve0: bigint, reserve1: bigint) => {
+  return {
+    currency0: WNATIVE[56],
+    currency1: WNATIVE[56],
```

Comment:
Let's use `currencyX` and `currencyY` to keep the naming aligned with smart contract